CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+## Update image gathering scripts
+
+For every release, we'll need to update the image gather scripts for fetching the images used by Knative Charms.
+
+To get an overview of how the script works please read https://github.com/canonical/knative-operators/issues/142
+The script has a limitation on how it's deducing images from upstream manifests, which is tracked in
+https://github.com/canonical/knative-operators/issues/220
+
+The script also needs to gather images for `net-istio`, that has it's own release cadance (although almost 1-1 with Knative Serving, but sometimes tags for Serving might not exist for `net-istio`).
+
+**Process:**
+1. Confirm the Knative Serving version you want to update `net-istio` for
+2. Deduce the `net-istio` that should be used
+  * If you update based on a Kubeflow release, then you can deduce this from [upstream Kubeflow manifests](https://github.com/kubeflow/manifests/blob/v1.9-branch/common/knative/README.md?plain=1#L8) (make sure to checkout to correct release branch)
```

Comment:
Should we tell readers what to look for in case the Line changes? Something like "Look for `Knative ingress controller for Istio ` in the README".

Reply:
Good point!

Reply:
Pushed a commit for this 
https://github.com/canonical/knative-operators/pull/221/commits/12bf44b5b3c3fbbd953ad1f9af26f0113eb5de0e